ddtrace/propagation/http.py

```diff
@@ -1068,7 +1139,17 @@ def my_controller(url, headers):
                     context = HTTPPropagator._resolve_contexts(contexts, styles_w_ctx, normalized_headers)
                     if config.propagation_http_baggage_enabled is True:
                         _attach_baggage_to_context(normalized_headers, context)
-                    return context
+
+            # baggage
```

Comment:
```suggestion
```
If it's important to mark this code block as "baggage-related", consider doing so by putting it into a meaningfully-named function rather than using a comment, which will become outdated.